vip-core/src/main/java/fr/insalyon/creatis/vip/core/client/view/common/MessageWindow.java

```diff
@@ -123,6 +124,7 @@ private void createLabel(String title, String icon) {
             public void onDraw(DrawEvent event) {
                 messageLabel.setWidth(messageLabel.getVisibleWidth());
                 messageLabel.setMargin(5);
+                messageLabel.setCanSelectText(true);
```

Comment:
This is called 3 times in this class. Only 1 must be enough.
I think only this call should be kept, on it would be better to call it above after `setContents`